### ISSUE TITLE
[WASM][WinUI3] ScrollViewer.ViewChanging is not implemented in Uno

### ISSUE THREAD

**Issue by ajpinedam** [unoplatform/uno#6746]
## Current behavior
Using DatePicker on a WASM / unoapp-winui app throws the following exception

![image](https://user-images.githubusercontent.com/1900897/129306350-e8c76f0f-12a8-404b-914b-e187f615e85c.png)

## How to reproduce it (as minimally and precisely as possible)
Install one of the latest Uno dotnet templates (for this example I used 3.10.0-dev.305)
Create a new app using dotnet new unoapp-winui
Add a DatePicker in the MainPage.xaml
Run WASM head
Click on the DatePicker

## Workaround
N/A

## Works on UWP/WinUI
Yes 

## Environment
WASM
Visual Studio

Nuget Package:
    <PackageReference Include="Microsoft.Windows.Compatibility" Version="5.0.0" />
    <PackageReference Include="Microsoft.Extensions.Logging" Version="5.0.0" />
    <PackageReference Include="Uno.Extensions.Logging.WebAssembly.Console" Version="1.0.1" />
    <PackageReference Include="Uno.WinUI.WebAssembly" Version="3.10.0-dev.305" />
    <PackageReference Include="Uno.WinUI.RemoteControl" Version="3.10.0-dev.305" Condition="'$(Configuration)'=='Debug'" />
    <PackageReference Include="Uno.Wasm.Bootstrap" Version="2.1.0" />
    <PackageReference Include="Uno.Wasm.Bootstrap.DevServer" Version="2.1.0" />

- [ ] Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia
- [x] Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia
- [ ] Uno.SourceGenerationTasks
- [ ] Uno.UI.RemoteControl / Uno.WinUI.RemoteControl
- [ ] Other: 

Nuget Package Version(s):

Affected platform(s):

- [ ] iOS
- [ ] Android
- [x] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [ ] macOS
- [ ] Skia
  - [ ] WPF
  - [ ] GTK (Linux)
  - [ ] Tizen
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

IDE:

- [ ] Visual Studio 2017 (version: )
- [x] Visual Studio 2019 (version: )
- [ ] Visual Studio for Mac (version: )
- [ ] Rider Windows (version: )
- [ ] Rider macOS (version: )
- [ ] Visual Studio Code (version: )

Relevant plugins:

- [ ] Resharper (version: )

**Comment by Youssef1313** [comment]
Is that a duplicate of https://github.com/unoplatform/uno/issues/78?

**Comment by ajpinedam** [comment]
@Youssef1313  yes, it seems to be a duplicate. Please mark this issue to close in your PR. Thanks

**Comment by MartinZikmund** [comment]
This no longer causes exception
